pkg/scheduler/core/generic_scheduler.go

```diff
@@ -89,9 +89,7 @@ func (g *genericScheduler) findClustersThatFit(
 	var out []*clusterv1alpha1.Cluster
 	clusters := clusterInfo.GetReadyClusters()
 	for _, c := range clusters {
-		resMap := fwk.RunFilterPlugins(ctx, placement, resource, c.Cluster())
-		res := resMap.Merge()
-		if !res.IsSuccess() {
+		if result := fwk.RunFilterPlugins(ctx, placement, resource, c.Cluster()); !result.IsSuccess() {
```

Comment:
I know it is allowed by Golang that involve methods like `IsSuccess` by a `nil` receiver, but it looks creepy(nil.IsSuccess()).

Reply:
Updated as make sense to newcomers when reading codes.